ajax_select/registry.py

```diff
@@ -17,7 +17,6 @@ def load_channels(self):
         Called when loading the application. Cannot be called a second time,
         (eg. for testing) as Django will not re-import and re-register anything.
         """
-        self._registry = {}
```

Comment:
Is this actually important? It tended to wipe the registry between the autodiscovery and settings-based discovery (which failed tests).

Reply:
The settings based discovery is getting removed right now anyway. Support will be 1.8+ so it's all autodiscovery.